frontend/src/view/UserComment/UserComment.js

```diff
@@ -0,0 +1,20 @@
+import { CommentContainer } from './CommentContainer.styled';
+import { ImageContainer } from './ImageContainer.styled';
+import { CommentContentContainer } from './CommentContentContainer.styled';
+import { useState } from 'react';
+import { RestoreMergeStatusError } from 'lint-staged/lib/symbols';
+
+const UserComment = () => {
+  const [comment, setComment] = useState('');
+  const [userImageUrl, setUserImageUrl] = useState('');
+  const [authorID, setAuthorID] = useState('');
+
+  return (
+    <CommentContainer>
+      <ImageContainer>
+        <img src={userImageUrl} alt="user " />
+      </ImageContainer>
+      <CommentContentContainer></CommentContentContainer>
+    </CommentContainer>
+  );
+};
```

Comment:
1 plik styled.js.

Taka ciekawostka do pisania kodu, który jeszcze do CR nie idzie, to często na czas pracy wrzuca się style do pliku .js z komponentem i dopopiero potem eksportuje :))

=Ale to taka rada na przyspieszenie pisania, a nie że must have.